lib/containers/container_images.pm

```diff
@@ -66,7 +67,9 @@ sub build_and_run_image {
     assert_script_run "curl -f -v " . data_url('containers/index.html') . " > $dir/BuildTest/index.html";
 
     # At least on publiccloud, this image pull can take long and occasinally fails due to network issues
-    $builder->build($dir . "/BuildTest", "myapp", timeout => 600);
+    my $timeout = 600;
+    $timeout = 1200 unless (is_x86_64);    # Increased timeout on non-x86_64 architectures
```

Comment:
ternary operator here will be awesome

Reply:
Good point! I sometimes forget that perl has ternary operators.